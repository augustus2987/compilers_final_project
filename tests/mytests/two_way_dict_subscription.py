x = {1:True, 2:True, 3:False}
print x[2]

x[2] = 33
print x
