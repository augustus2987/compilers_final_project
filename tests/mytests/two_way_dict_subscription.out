True
{3: False, 1: True, 2: 33}
